src/style.css

```diff
@@ -1,3 +1,70 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+a {
+  text-decoration: none;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  justify-content: space-between;
+  width: 100%;
```

Comment:
You can also limit the `min-width` so that the `header `cannot become smaller than `1024px`